app/javascript/components/molecules/ExpandableTable.js

```diff
@@ -39,7 +39,7 @@ const ExpandableTableRow = ({ row, rowKeyIndex, numColumns, expanderRenderer, ex
 }
 
 const ExpandableTable = ({ columns, rows, rowKeyIndex, expanderRenderer, expandedRowRenderer, originals, classes, closeAllRows }) => (
-  <table className={classNames('td-light td-plain th-plain', classes)} role='grid'>
+  <table className={classNames('td-light td-plain th-plain', classes)} style={{ background: 'transparent' }} role='grid'>
```

Comment:
nit: can you move these props onto separate lines?

Reply:
do we want to have a standard # of props at which we move to their own lines? in theory our linter would complain about line length.

Reply:
Airbnb styleguide says more than 1 prop always wraps to a new line which I think is a little aggressive.

I think line length is is a good way to do it but it doesn't seem like standard.js complains about long lines, so that's something we can add when we get around to extending the standard linter.

If you want a hard and fast rule so you don't have to think about it maybe we say 3 props? Otherwise I just look at how complex the line is and decide that way 🤷